src/pages/Hwangsoohyeon/Join/Join.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import './Join.scss';
+import { useNavigate } from 'react-router-dom';
+
+const Join = () => {
+  const navigate = useNavigate();
+  const [inputid, setinputid] = useState('');
+  const [inputpw, setinputpw] = useState('');
+
+  const goToJoin_ok = () => {
+    navigate('/Join_ok');
+  };
+
+  const goToLogin = () => {
+    navigate('/Login');
+  };
+
+  const saveUserId = (e) => {
+    setinputid(e.target.value);
```

Comment:
아래와같이 value를 구조분해할당해서 사용할 수 있습니다
```suggestion
  const saveUserId = (e) => {
    const { value } = e.target
    setinputid(value);
```